google/cloud/accessapproval/quickstart/.bazelrc

```diff
@@ -17,6 +17,19 @@
 #   [1]: https://github.com/bazelbuild/bazel/issues/4341
 build --copt=-DGRPC_BAZEL_BUILD
 
+# Use host-OS-specific config lines from bazelrc files.
+build --enable_platform_specific_config=true
+
+# The project requires C++ >= 14. By default Bazel adds `-std=c++0x` which
+# disables C++14 features, even if the compilers defaults to C++ >= 14
+# Use host-OS-specific config lines from bazelrc files.
+build --enable_platform_specific_config=true
```

Comment:
Duplication.

Reply:
Thanks.  I found that in only two files, and fixed both.